### PR TITLE
chore(create-farm): update template version

### PR DESCRIPTION
**Pull request by ErKeLost**
**Description:**

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: packages/create-farm/templates/react/package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "react-dom": "18"
   },
   "devDependencies": {
-    "@farmfe/cli": "^0.5.1",
-    "@farmfe/core": "^0.6.4",
+    "@farmfe/cli": "^0.5.2",
+    "@farmfe/core": "^0.7.6",
     "@farmfe/plugin-react": "0.1.3",
```
Comment (wre232114) [inline packages/create-farm/templates/react/package.json:12]: the latest version of plugin-react is 0.1.5

File: packages/create-farm/templates/vue/package.json [hunk 2]
```diff
@@ -7,12 +7,13 @@
     "core-js": "^3.30.1"
   },
   "devDependencies": {
-    "@farmfe/cli": "^0.5.1",
-    "@farmfe/core": "^0.6.4",
+    "@farmfe/cli": "^0.5.2",
+    "@farmfe/core": "^0.7.6",
     "@farmfe/js-plugin-vue": "^0.2.1"
```
Comment (wre232114) [inline packages/create-farm/templates/vue/package.json:12]: latest version is 1.0.1

Comment (ErKeLost) [reply packages/create-farm/templates/vue/package.json]: It's done.

File: packages/create-farm/templates/react/package.json (hunk 1 above)
Comment (ErKeLost) [reply packages/create-farm/templates/react/package.json]: It's done.

**Pull request merged**